### ISSUE TITLE
Add an open code list for role on Membership and Post, for classification on Organization and Event

### ISSUE THREAD

**Issue by jpmckinney** [popolo-project/popolo-spec#91]
Membership/Post role
- member (default)
- candidate
- deputy

Organization classification
- jurisdiction, e.g. "Ville de Montréal"
- council, e.g. "Conseil municipal de la Ville de Montréal"
- committee, e.g. "Comité exécutif de la Ville de Montréal"
- legislature, e.g. "United States Congress"
- chamber, e.g. "United States House of Representatives"
- political party, e.g. "Rhinoceros Party"

Event classification
- legislature
- session
- sitting

Add aliases to the appropriate JSON-LD contexts.

**Comment by akuckartz** [comment]
`skos:Concept`s in JSON-LD ;-)

**Comment by jpmckinney** [comment]
Yes, the hard part is determining which concepts to include, which is what the issue is about :)

**Comment by akuckartz** [comment]
The concepts do not need to be centralized (be specified in the specification) !

**Comment by jpmckinney** [comment]
Yes, but there is value in people using the same concepts and not constantly inventing new concepts that don't relate to each other, so it makes sense provide a list (or ConceptScheme in SKOS terminology).

**Comment by kaerumy** [comment]
Could we add some more to the organization classification?
- statutory corporation eg. Petronas (National Oil Corporation)
- public limited company eg. Nike Corporation
- private limited company eg. Dell Inc.
- foundation eg. Knight Foundation
- board eg. Board of Directors, Board of Advisers, Board of Trustees etc.
- union eg. Construction Workers Union

Why would we want to do this for gov specifications? Cause in Malaysia we have government owned companies in which the executive sits on company boards, so it is still needed.
